Index: src/main/java/org/example/Bank.java
===================================================================
diff --git a/src/main/java/org/example/Bank.java b/src/main/java/org/example/Bank.java
--- a/src/main/java/org/example/Bank.java	
+++ b/src/main/java/org/example/Bank.java	
@@ -1,12 +1,12 @@
 package org.example;
 
 public class Bank {
-    static String bankName;
+    String bankName;
     int creditLimit;
     double fee;
 
     public Bank() {
-        this.bankName = "Bank";
+        this.bankName = "Bank"; // TODO: заменить на параметры
         this.creditLimit = 1000000;
         this.fee = 0.05;
     }
@@ -14,11 +14,11 @@
         return 0;
     }
     public Debit debit(){
-        Debit debit = new Debit();
+        Debit debit = new Debit();//TODO: аргументы
         return debit;
     }
     public Credit credit(){
-        Credit credit = new Credit();
+        Credit credit = new Credit();//TODO: аргументы
         return credit;
     }
 }
Index: src/main/java/org/example/Debit.java
===================================================================
diff --git a/src/main/java/org/example/Debit.java b/src/main/java/org/example/Debit.java
--- a/src/main/java/org/example/Debit.java	
+++ b/src/main/java/org/example/Debit.java	
@@ -2,6 +2,11 @@
 
 public class Debit extends Account {
 
+    public Debit(int balance, String login, String bankName, String accountType) {
+        super(balance, login, bankName, accountType);
+    }
+
+
     @Override
     public void push(int sum) {
 
Index: src/main/java/org/example/Main.java
===================================================================
diff --git a/src/main/java/org/example/Main.java b/src/main/java/org/example/Main.java
--- a/src/main/java/org/example/Main.java	
+++ b/src/main/java/org/example/Main.java	
@@ -18,11 +18,11 @@
             System.out.println("Password: ");
             String password = scanner.next();
             if (!password.equals(currentClient.password)) {
-                System.out.println("Access denied");
+                System.out.println("Access denied"); //while? while do?
                 System.exit(1);
             }
 
-        } else if (sign.equals("sign up")) {
+        } else if (sign.equals("sign up")) { //client constructor, move to client class
             System.out.println("Name: ");
             String name = scanner.next();
             System.out.println("Surname: ");
@@ -47,7 +47,7 @@
                 System.out.println(element);
             }
             System.out.println("Choose account or create a new one");
-            System.out.println("If you want to choose any account enter it's name from the list");
+            System.out.println("If you want to choose any account enter it's name from the list");//TODO: обработка сценария, прописать действия
         } else {
             System.out.println("You have not got any accounts yet");
         }
@@ -57,16 +57,17 @@
             String bankName = scanner.next();
             System.out.println("Choose account type (debit/credit): ");
             String accountType = scanner.next();
+            Account currAccount = null;
             if (accountType.equals("debit")) {
-                Debit currAccount = new Debit();
+                currAccount = new Debit();//variables
             } else if (accountType.equals("credit")) {
-                Credit currAccount = new Credit();
+                 currAccount = new Credit();//variables
             } else {
                 System.out.println("Error");
                 System.exit(1);
             }
 
-            //String currID = currAccount.setID();
+            String currID = currAccount.setID();//variables
 
         }
     }
Index: src/main/java/org/example/Base.java
===================================================================
diff --git a/src/main/java/org/example/Base.java b/src/main/java/org/example/Base.java
--- a/src/main/java/org/example/Base.java	
+++ b/src/main/java/org/example/Base.java	
@@ -1,8 +1,15 @@
 package org.example;
+import org.json.simple.JSONObject;
+import org.json.simple.JSONArray;
 
 import java.util.HashMap;
+import java.util.Map;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
 
 public class Base {
+    Map<String, Object> mapping = new ObjectMapper().readValue(jsonStr, HashMap.class);
     static HashMap<String, Client> base = new HashMap<>();
     static HashMap<String,Account> accountBase  = new HashMap<>();
 }
Index: src/main/java/org/example/AbstractClient.java
===================================================================
diff --git a/src/main/java/org/example/AbstractClient.java b/src/main/java/org/example/AbstractClient.java
--- a/src/main/java/org/example/AbstractClient.java	
+++ b/src/main/java/org/example/AbstractClient.java	
@@ -1,5 +1,6 @@
 package org.example;
 
+import java.util.ArrayList;
 import java.util.LinkedList;
 
 abstract class AbstractClient {
@@ -9,7 +10,7 @@
     String passport;
     String login;
     String password;
-    LinkedList<String> accounts;
+    ArrayList<String> accounts;
 
     //ID
     public int findLogin(String userName, String userSurname) {
@@ -18,7 +19,7 @@
 
     ;public AbstractClient(String userName, String userSurname,
                            String address, String passport,
-                           String login, String password, LinkedList<String> accounts){
+                           String login, String password, ArrayList<String> accounts){
         this.userName = userName;
         this.userSurname = userSurname;
         this.address = address;
